README.md

```diff
@@ -38,6 +38,18 @@ Getting Started
 <dependencies>
     <dependency>
         <groupId>com.simple</groupId>
+        <artifactId>simplespec_2.12.0</artifactId>
```

Comment:
this doc is actually wrong. we use `major.version` only, so:

```xml
<artifactId>simplespec_2.12</artifactId>
```

Also, if you could update line 53, that would be good.